### PR TITLE
f-footer@2.1.0 - Fix country selector closing on-click for mobile

### PR DESCRIPTION
**Pull request by laurenrodney**
Moved v-click-outside handler from button to parent element to prevent the country selector from closing on-click

---

## UI Review Checks

- [x] This code has been checked with regard to [our accessibility standards](http://fozzie.just-eat.com/documentation/general/accessibility/checklist)

## Browsers Tested

- [x] Chrome (latest)
- [ ] Internet Explorer 11
- [x] Mobile (iPhone and Android on chrome)

### CODE REVIEW

**Review by DamianMullins — DISMISSED** [review]

**Review by simonsandersje — DISMISSED** [review]

**Review by ashleynolan — DISMISSED** [review]
Got conflicts to resolve, but otherwise LGTM :shipit:

File: packages/f-footer/CHANGELOG.md
Comment (ashleynolan) [inline packages/f-footer/CHANGELOG.md]: You can delete this line and the one below now (think you had before, but the merge has brought it back)

**Review by ashleynolan — DISMISSED** [review]

**Review by asherstoppard — DISMISSED** [review]

**Review by xander-marjoram — CHANGES_REQUESTED** [review]
Looks like we just need to fix the changelog so we're not writing over Simon's entry

**Review by xander-marjoram — APPROVED** [review]

**Review by ashleynolan — APPROVED** [review]

**Pull request merged**
